src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {},
```

Comment:
Nice job so far👍🏼. It is a requirement that a reducer that checks status and always returns "under construction" is implemented. kindly do well to fix this👍🏼.

<img width="973" alt="image" src="https://github.com/78fatemanazari/react-bookstore/assets/64643995/ee6d7912-299a-4a3d-b54e-1db9664b8b5a">